[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.36.248.235:9200/series_index/_search_shards
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.36.248.29:9200/
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.36.248.29:9200/
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.36.248.29:9200/
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.36.248.29:9200/
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.36.248.235:9200/series_index/_search_shards
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:329 



[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:330 Model  Controller           Cloud/Region         Version  SLA          Timestamp
test   localhost-localhost  localhost/localhost  3.5.3    unsupported  01:51:47Z

App                       Version  Status  Scale  Charm                     Channel        Rev  Exposed  Message
opensearch                         active    2/3  opensearch                                 0  no       
self-signed-certificates           active      1  self-signed-certificates  latest/stable  155  no       

Unit                         Workload  Agent  Machine  Public address  Ports     Message
opensearch/0                 unknown   lost   1        10.36.248.29    9200/tcp  agent lost, see 'juju show-status-log opensearch/0'
opensearch/1                 active    idle   2        10.36.248.114   9200/tcp  
opensearch/2*                active    idle   3        10.36.248.235   9200/tcp  
self-signed-certificates/0*  active    idle   0        10.36.248.218             

Machine  State    Address        Inst id        Base          AZ  Message
0        started  10.36.248.218  juju-9cb857-0  ubuntu@22.04      Running
1        down     10.36.248.29   juju-9cb857-1  ubuntu@22.04      Running
2        started  10.36.248.114  juju-9cb857-2  ubuntu@22.04      Running
3        started  10.36.248.235  juju-9cb857-3  ubuntu@22.04      Running

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:51:47 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.126  -- (10.36.248.29) -- [lost (since: 01:47:06)] unknown: agent lost, see 'juju show-status-log opensearch/0'
		opensearch-1.126  -- (10.36.248.114) -- [idle (since: 01:49:11)] active: 
		opensearch-2.126* -- (10.36.248.235) -- [idle (since: 01:51:42)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:52:17 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.126  -- (10.36.248.29) -- [executing (since: 01:52:13)] maintenance: Requesting new TLS certificates...
		opensearch-1.126  -- (10.36.248.114) -- [executing (since: 01:52:11)] active: 
		opensearch-2.126* -- (10.36.248.235) -- [executing (since: 01:52:16)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:52:28 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.126  -- (10.36.248.29) -- [executing (since: 01:52:13)] waiting: The OpenSearch service is stopping.
		opensearch-1.126  -- (10.36.248.114) -- [idle (since: 01:52:26)] active: 
		opensearch-2.126* -- (10.36.248.235) -- [executing (since: 01:52:16)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:52:40 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.126  -- (10.36.248.29) -- [executing (since: 01:52:13)] waiting: The OpenSearch service stopped.
		opensearch-1.126  -- (10.36.248.114) -- [idle (since: 01:52:26)] active: 
		opensearch-2.126* -- (10.36.248.235) -- [idle (since: 01:52:34)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:52:52 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.126  -- (10.36.248.29) -- [executing (since: 01:52:13)] waiting: Waiting for OpenSearch to start...
		opensearch-1.126  -- (10.36.248.114) -- [idle (since: 01:52:26)] active: 
		opensearch-2.126* -- (10.36.248.235) -- [idle (since: 01:52:34)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:53:04 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.126  -- (10.36.248.29) -- [executing (since: 01:53:02)] waiting: Waiting for OpenSearch to start...
		opensearch-1.126  -- (10.36.248.114) -- [executing (since: 01:53:02)] active: 
		opensearch-2.126* -- (10.36.248.235) -- [executing (since: 01:53:02)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:53:15 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.126  -- (10.36.248.29) -- [executing (since: 01:53:02)] waiting: Waiting for OpenSearch to start...
		opensearch-1.126  -- (10.36.248.114) -- [idle (since: 01:53:15)] active: 
		opensearch-2.126* -- (10.36.248.235) -- [idle (since: 01:53:15)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:53:26 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.126  -- (10.36.248.29) -- [executing (since: 01:53:02)] waiting: Waiting for OpenSearch to start...
		opensearch-1.126  -- (10.36.248.114) -- [idle (since: 01:53:15)] active: 
		opensearch-2.126* -- (10.36.248.235) -- [idle (since: 01:53:15)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:53:38 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.126  -- (10.36.248.29) -- [idle (since: 01:53:34)] active: 
		opensearch-1.126  -- (10.36.248.114) -- [idle (since: 01:53:31)] active: 
		opensearch-2.126* -- (10.36.248.235) -- [idle (since: 01:53:32)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:53:49 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.126  -- (10.36.248.29) -- [idle (since: 01:53:34)] active: 
		opensearch-1.126  -- (10.36.248.114) -- [idle (since: 01:53:31)] active: 
		opensearch-2.126* -- (10.36.248.235) -- [idle (since: 01:53:32)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:54:01 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.126  -- (10.36.248.29) -- [idle (since: 01:53:34)] active: 
		opensearch-1.126  -- (10.36.248.114) -- [idle (since: 01:53:31)] active: 
		opensearch-2.126* -- (10.36.248.235) -- [idle (since: 01:53:32)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:54:13 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.126  -- (10.36.248.29) -- [idle (since: 01:53:34)] active: 
		opensearch-1.126  -- (10.36.248.114) -- [idle (since: 01:53:31)] active: 
		opensearch-2.126* -- (10.36.248.235) -- [idle (since: 01:53:32)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:54:24 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.126  -- (10.36.248.29) -- [idle (since: 01:53:34)] active: 
		opensearch-1.126  -- (10.36.248.114) -- [idle (since: 01:53:31)] active: 
		opensearch-2.126* -- (10.36.248.235) -- [idle (since: 01:53:32)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:54:36 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:277 	App: opensearch - Units - conditions unmet.
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:278 
	app: opensearch active -- message: None
		opensearch-0.126  -- (10.36.248.29) -- [idle (since: 01:53:34)] active: 
		opensearch-1.126  -- (10.36.248.114) -- [idle (since: 01:53:31)] active: 
		opensearch-2.126* -- (10.36.248.235) -- [idle (since: 01:53:32)] active: 

[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:337 


01:54:47 -- Waiting for model...
[32mINFO    [0m integration.helpers_deployments:helpers_deployments.py:348 01:54:49 -- Waiting for model: complete.



[31m[1mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[31m[1mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[31m[1mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[31m[1mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[31m[1mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[31m[1mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[31m[1mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[31m[1mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[31m[1mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[31m[1mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[31m[1mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[31m[1mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[31m[1mERROR   [0m integration.ha.helpers:helpers.py:409 Network not restored yet, attempting again.
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.36.248.96:9200/
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.36.248.235:9200/series_index/_search_shards
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.36.248.96:9200/_nodes
[32mINFO    [0m integration.ha.helpers:helpers.py:247 Continuous writes result: namespace(max_stored_id=927, count=928, last_expected_id=927)
[32mINFO    [0m integration.helpers:helpers.py:347 Calling: GET -- https://10.36.248.235:9200/series_index/_search_shards
[32mINFO    [0m integration.ha.conftest:conftest.py:48 



The writes have been cleared.